app/frontend/src/features/messages/utils.ts

```diff
@@ -0,0 +1,43 @@
+import { Message } from "../../pb/conversations_pb";
+import { hostRequestStatusLabels } from "./constants";
+
```

Comment:
Are there plans to use these helpers for communities chat as well since you got these in its own file?

Reply:
No I just didn't think it fit well in utile. It will also be used for buttons to set the status when responding